docs/guides/vault/restrict-addresses-to-vaults.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_label: 'Restrict Addresses to Vaults'
+sidebar_position: 4
+---
+
+# Restrict Addresses to Vaults
+
+As mentioned in the [first Vault guide](./create-a-vault.md), the **Vault** can be used to restrict different addresses (protocols, other devices, etc..) to execute and set data on it instead of setting and executing directly on the Universal Profile. 
```

Comment:
```suggestion
As mentioned in the [first Vault guide](./create-a-vault.md), the **Vault** can be used to restrict different addresses (protocols, other devices, etc.) to execute and set data on it instead of doing it directly on the Universal Profile. 
```